osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/DeCoupleMBusDeviceCommandExecutorTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.mbus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.openmuc.jdlms.MethodResultCode;
+import org.openmuc.jdlms.ObisCode;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils.CosemObjectAccessor;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.DlmsDevice;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.factories.DlmsConnectionManager;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.ChannelElementValuesDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.DeCoupleMbusDeviceDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.DeCoupleMbusDeviceResponseDto;
+
+@ExtendWith(MockitoExtension.class)
+public class DeCoupleMBusDeviceCommandExecutorTest {
+
+    @Mock
+    private DeviceChannelsHelper deviceChannelsHelper;
+
+    @Mock
+    private DlmsConnectionManager conn;
+
+    @Mock
+    private DlmsDevice device;
+
+    @Mock
+    private DeCoupleMbusDeviceDto decoupleMbusDto;
+
+    @InjectMocks
+    private DeCoupleMBusDeviceCommandExecutor commandExecutor = new DeCoupleMBusDeviceCommandExecutor();
+
+    @Test
+    public void test() throws ProtocolAdapterException {
```

Comment:
I assume test() is the happy flow? consider renaming to the case you are testing here.

Reply:
testHappyFlow()